### PR TITLE
Stored procedure to change fin period (incl year) in Solar Mock DB

### PR DESCRIPTION
**Pull request by frankvanrietimqs**

### CODE REVIEW

**Review by DFRootman — APPROVED** [review]
LGTM

File: scripts/solar/stored-procs/09-upd-finperiod.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+AS
+BEGIN
+	UPDATE Budget SET FINANCIAL_YEAR = @NewYear;
+	UPDATE ProjectSCOASegment SET FINANCIAL_YEAR = @NewYear;
+	UPDATE Item SET FINANCIAL_YEAR = @NewYear;
+
+	UPDATE OpenPeriods SET 
+		FINANCIAL_YEAR = @NewYear,  
+		PERIOD_STATUS = (SELECT DESCRIPTION FROM PeriodStatus WHERE CODE = 1),
+		PERIOD_STATUS_CODE = 1
```
Comment (navonod) [inline scripts/solar/stored-procs/09-upd-finperiod.sql]: you can remove this line, as no change in value will ever occur.

**Review by navonod — COMMENTED** [review]
Looks ok - don't you also want to be doing this in the SCOAJournal though?

File: scripts/solar/stored-procs/09-upd-finperiod.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
+-- This is a utility method for our mock setups that allows us to switch between financial years
+CREATE PROCEDURE [dbo].[P_F_G_UPD_FIN_PERIOD] (@NewYear VARCHAR(4), @NewPeriod VARCHAR(2))
+AS
+BEGIN
+	UPDATE Budget SET FINANCIAL_YEAR = @NewYear;
+	UPDATE ProjectSCOASegment SET FINANCIAL_YEAR = @NewYear;
+	UPDATE Item SET FINANCIAL_YEAR = @NewYear;
+
+	UPDATE OpenPeriods SET
```
Comment (nzali) [inline scripts/solar/stored-procs/09-upd-finperiod.sql:9]: Forgot to set FINANCIAL_PERIOD perhaps?

**Pull request merged**